public/index.html

```diff
@@ -5,8 +5,9 @@
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
-    <meta name="description" content="Covid-19-AU" />
-    <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo.jpg" />
+    <meta name="description" content="Track Coronavirus cases in Australia with trends, analytics and state-by-state breakdowns. 
```

Comment:
maybe the sentence could start with  "_Track Coronavirus (covid-19) cases...._"

Reply:
Made a new commit with this @freddiewanah. I also reduced the character count to fit Google's cap of 160.